back/engines/commercial/bulk_import_ideas/app/controllers/bulk_import_ideas/web_api/v1/import_ideas_controller.rb

```diff
@@ -26,16 +26,24 @@ def example_xlsx
       send_data xlsx, type: 'application/vnd.openxmlformats-officedocument.spreadsheetml.sheet', filename: 'ideas.xlsx'
     end
 
-    def draft_ideas
-      creation_phase_id = nil
-      if import_scope == :phase
-        phase = Phase.find(params[:id])
-        creation_phase_id = phase&.native_survey? ? phase.id : nil
+    def approve_all
+      ideas = imported_draft_ideas
+      approved = 0
+      not_approved = 0
+      ideas.each do |idea|
+        if idea.update(publication_status: 'published')
+          approved += 1
+        else
+          not_approved += 1
+        end
       end
-      ideas = Idea.draft.where(project_id: @project.id, creation_phase_id: creation_phase_id)
 
+      render json: raw_json({ approved: approved, notApproved: not_approved })
```

Comment:
```suggestion
      render json: raw_json({ approved: approved, not_approved: not_approved })
```
I know it's a bit awkward to convert from camel case to snake case on the FE, but maybe we should still stick to our convention? As a bonus, you can use this nice Ruby 3.1 syntax :slightly_smiling_face: 

```suggestion
      render json: raw_json({ approved:, not_approved: })
```